intermediate_html_css/intermediate_html_concepts/svgs.md

```diff
@@ -8,39 +8,43 @@ In this lesson, we will learn exactly what SVGs are, what they're used for, and
 
 SVGs are often used for:
 
-1.  Icons
-2.  Graphs/Charts
-3.  Large, simple images
-4.  Patterned backgrounds
-5.  Applying effects to other elements via SVG filters
+1. Icons
+1. Graphs/Charts
+1. Large, simple images
+1. Patterned backgrounds
+1. Applying effects to other elements via SVG filters
 
 ### Okay, but what are they?
 
-"SVG" stands for "Scalable Vector Graphics". Vector graphics are images defined by math, as opposed to traditional "raster graphics", where your image is defined by a grid of pixels. With raster graphics, the detail is limited to the size of that pixel grid. If you want to increase the size of the image (_scale_ it), you have to increase the size of that grid. How do you decide what all those new pixels should look like? There's no simple solution. Additionally, the larger the grid, the bigger your filesize grows.
+"SVG" stands for "Scalable Vector Graphics". Vector graphics are images defined by math, as opposed to traditional "raster graphics", where your image is defined by a grid of pixels. With raster graphics, the detail is limited to the size of that pixel grid. If you want to increase the size of the image (*scale* it), you have to increase the size of that grid. How do you decide what all those new pixels should look like? There's no simple solution. Additionally, the larger the grid, the bigger your filesize grows.
 
 With vector graphics on the other hand, there's no grid. Instead, you have formulas for different shapes and lines. Since these are just formulas, it doesn't matter how large or small you want them to appear--they can scale to any size you want, and it will have no effect on the quality or the size of the file.
 
 SVGs have another interesting aspect to them: they're defined using XML. XML (aka, "Extensible Markup Language") is an HTML-like syntax which is used for lots of things, from [APIs](https://en.wikipedia.org/wiki/API), to [RSS](https://en.wikipedia.org/wiki/RSS), to [spreadsheet and word editor software](https://en.wikipedia.org/wiki/Office_Open_XML).
 
 The fact that SVG source-code is XML has a few key benefits.
 
-First, it means that it is _human-readable_. If you were to open up a JPEG in a text editor, it would just look like gobbledygook. If you were to open up an SVG, however, it would look something like this:
+First, it means that it is *human-readable*. If you were to open up a JPEG in a text editor, it would just look like gobbledygook. If you were to open up an SVG, however, it would look something like this:
 
 ```xml
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+
     <rect x=0 y=0 width=100 height=50 />
+
     <circle class="svg-circle" cx="50" cy="50" r="10"/>
+
 </svg>
+
 ```
```

Comment:
````suggestion
```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
  <rect x=0 y=0 width=100 height=50 />
  <circle class="svg-circle" cx="50" cy="50" r="10"/>
</svg>
```
````
Mind reverting the new line stuff and changing the code block language to `html`?
The linter won't complain if it's inside an HTML block. *Technically* `xml` is more correct but `html` is perfectly fine in context and this is the sole occurrence of an `xml` code block in the entire curriculum, so not worth adding to the TOP005 rule exceptions just yet.